README.md

```diff
@@ -73,6 +83,55 @@ You'll need a trello account and fill in your trello credentials in the .env fil
 
 Please ensure that you populate the `.env` file with the details of names and IDs of your respective lists within Trello. 
 
+### ** Testing **
 
+The application has been built with some standard tests to ensure that the everything is fully working.  In order to use these, you need to run the following python test files 
+
+``` 
+$ python test_view_model.py
+```
+This is the main file that contains all of the tests.  Running this python file will run all tests on the application.  
 
+The following tests will also ensure the application is fully working prior to use.  They will test the functionality around the tasks within the application.  
+```
+* test_to_do
+```
+Tests and confirms the status of all tasks not yet started.  
 
+```
+* test_doing
+```
+Tests and confirms the status of all tasks that are currently In Progress.
+```
+* test_done
+```
+Tests and confirms the status of all tasks that are competed.  
+```
+* test_show_all_done_items_returns_for_small_numbers_of_items
+```
+This tests the functionality about returning a shortened list of either 5 or less completed items.  This will support the different view options within the application.  
+```
+* test_show_all_done_items_returns_false_for_large_numbers_of_items
+```
+This test will confirm the function of returning results for completed tasks whereby the quantity if greater than 5.  This will also support differnt view options within the application.  
+
+ 
+### Docker
+
+To create a Docker production image, use the following;
+
+```
+docker build --target production --tag todo-app:prod .
+```
+
+To create a Docker development image, use the following;
+
+```
+docker build --target production --tag todo-app:dev .
+```
+
+Use the following command to pass the key variables into the container.  
+
+```
+docker run --env-file .env -p 5000:5000 todo-app 
```

Comment:
You're missing one of the requirements mentioned in the instructions.

If you run with a bind mount, then you can update files and see that change appear in the container without having to rebuild the image or even restart the container.

Depending on your terminal, you want something like: `--mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app`